resources/views/service/printables/families.blade.php

```diff
@@ -31,7 +31,12 @@
         </tr>
         @foreach ($registrations as $registration)
         <tr>
-          <td>{{ $registration->family->pri_carer }}</td>
+          <td>
+            @if($registration->family->getNoticeReasons())
```

Comment:
hmm. try count($registration->family->getNoticeReasons() > 0) ?
or explicitly check the array is not empty.

Reply:
![screen shot 2018-02-26 at 16 43 22](https://user-images.githubusercontent.com/11891294/36682960-4764f9b4-1b14-11e8-9708-22942fec5934.png)
doesnt look like it works as it applies the 'a' to all families. Will have to have a proper look tomorrow @charlesstrange2